recipes/imagemagick/all/conandata.yml

```diff
@@ -1,13 +1,19 @@
 sources:
+  "7.1.0-45":
+    url: "https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.0-45.tar.gz"
+    sha256: "3df6ca6dff15a4e8a20b4593c60285a59e38890440494d91a344e5c0e2bb3eec"
   "7.0.11-14":
-    source:
-      url: "https://github.com/ImageMagick/ImageMagick/archive/7.0.11-14.tar.gz"
-      sha256: "dfa5aa3f7f289f12c2f9ee6c7c19b02ae857b4eec02f40298f60f5c11048a016"
-    visualmagick:
-      "fcd198e4344576b9efc2facd952a2f0c57752c07":
-        url: "https://github.com/ImageMagick/VisualMagick/archive/fcd198e4344576b9efc2facd952a2f0c57752c07.zip"
-        sha256: "f793c1c48796ed2f295ba9cacb938f6aa7e4bf011226d96f977655911852bfda"
+    url: "https://github.com/ImageMagick/ImageMagick/archive/7.0.11-14.tar.gz"
+    sha256: "dfa5aa3f7f289f12c2f9ee6c7c19b02ae857b4eec02f40298f60f5c11048a016"
+
 patches:
   "7.0.11-14":
-    - patch_file: "patches/remove-unnecessary-dependencies.patch"
-      base_path: "VisualMagick"
+    - patch_file: "patches/fix_linking_against_libtiff.patch"
+  "7.1.0-45":
+    - patch_file: "patches/disable_complex_fftw.patch"
```

Comment:
Please, documented those new patches following patch_description and patch_type. If they are consumed from the upstream, please add patch_source too.

More info: https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields